Jenkinsfile

```diff
@@ -70,6 +70,34 @@ node('test') {
                 junit 'target/junit/TEST-Jest Integration Tests*.xml'
             }
 
+            stage('Plugin Functional Test') {
+                currentBuild.result = 'Success'
+                echo "Start Plugin Functional Test"
+                echo "TEST_BROWSER_HEADLESS $env.TEST_BROWSER_HEADLESS"
+                echo "NODE_OPTIONS $env.NODE_OPTIONS"
+
+                withEnv([
+                    "TEST_BROWSER_HEADLESS=1",
+                    "CI=1",
+                    "TEST_ES_PORT=9200",
+                    "TEST_KIBANA_PORT=5601",
+                    "TEST_KIBANA_PROTOCOL=http",
+                    "TEST_ES_PROTOCOL=http",
+                    "TEST_KIBANA_HOSTNAME=localhost",
+                    "TEST_ES_HOSTNAME=localhost"
+                ]) {
+
+                def pluginFtrResult = sh returnStatus: true, script: "CI=1 GCS_UPLOAD_PREFIX=fake node scripts/functional_test_runner.js --config test/plugin_functional/config.js"
+
+                if (pluginFtrResult != 0) {
+                    currentBuild.result = 'FAILURE'
+                }
+
+                junit 'target/junit/TEST-Plugin Functional Tests*.xml'
```

Comment:
Is the tests using the same name prefix? or could be change later?

Reply:
Are you referring to `GCS_UPLOAD_PREFIX=fake`?

Reply:
I'm referring the name "**TEST-Plugin Functional Tests***.xml"

Reply:
This line is telling junit where in our workspace to place & retrieve the test report to/from. Allowing us to see the individual test results in blue ocean gui. Why would you want to change it?

Reply:
I thought that original is **/*.xml? or maybe I'm wrong here.

Reply:
The test files structure doesn't need to change by our project